### PR TITLE
Update appveyor config

### LINKED ISSUE #369: Fix Appveyor builds
Since the organisation name changed the Appveyor builds are broken. Will probably need to set them up again. Check the URL for badge in the README also.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Closes #369

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Looks like some transient problem with appveyor/conda. Will try again later today.

**Comment by jeromekelleher** [comment]
Still not working, but it definitely looks like a problem on the appveyor side. Will try again tomorrow.

**Comment by grahamgower** [comment]
I think the appveyor problem is described in the conda troubleshooting guide.
https://conda.io/projects/conda/en/latest/user-guide/troubleshooting.html#ssl-connection-errors

**Comment by jeromekelleher** [comment]
> I think the appveyor problem is described in the conda troubleshooting guide.
> https://conda.io/projects/conda/en/latest/user-guide/troubleshooting.html#ssl-connection-errors

Thanks @grahamgower, good spot. I tried adding `` conda config --set ssl_verify false`` but it didn't help. The weird thing here is that basically exactly the same Appveyor config is working fine on tskit...

**Comment by grahamgower** [comment]
I don't think its a lack of appropriate certificates (which might be fixed by setting ssl_verify=false), it looks more like the openssl libs are not being found. It might need some %PATH% frobbing or a magic `conda activate` command.

I suppose there's some version differences between the windows CI here and for tskit?

**Comment by jeromekelleher** [comment]
> I don't think its a lack of appropriate certificates (which might be fixed by setting ssl_verify=false), it looks more like the openssl libs are not being found. It might need some %PATH% frobbing or a magic conda activate command.

Yes, you're right. I'll have another look.

> I suppose there's some version differences between the windows CI here and for tskit?

Must be, I guess.

**Comment by jeromekelleher** [comment]
It worked! Thanks @grahamgower, you saved me a lot of time there.

Now we just need to fixup the tests broken on windows. I'll just skip any that seem tricky and fix easy ones.

**Comment by jeromekelleher** [comment]
@grahamgower, I made some minimal changes to get this to work on Windows. Only potentially controversial thing is to not add slim as an engine on Windows. Do you see any potential issues with this?

**Comment by grahamgower** [comment]
The command line options for slim will still be shown. And all the slim tests need to be skipped.

I think getting the slim engine to work on windows will not be a lot of work though, it probably just needs (1) to use a temp directory instead of temp files, and maybe (2) get_version() should not fail if the slim binary is not available. Of course, developing for windows without a windows machine is a pain.

**Comment by jeromekelleher** [comment]
> The command line options for slim will still be shown. And all the slim tests need to be skipped.

Good call. How does the latest update look?

**Comment by jeromekelleher** [comment]
> I think getting the slim engine to work on windows will not be a lot of work though, it probably just needs (1) to use a temp directory instead of temp files, and maybe (2) get_version() should not fail if the slim binary is not available. Of course, developing for windows without a windows machine is a pain.

We'd need SLiM to work on windows first though. Until this is supported upstream there's no point in us putting effort in. (I've been nagging Ben about this, but I think he needs actual windows users to start making their voices heard. No reason SLiM couldn't work natively on windows.)

**Comment by jeromekelleher** [comment]
Codecov reports are misleading here, can be ignored.

File: stdpopsim/slim_engine.py [hunk 1]
```diff
@@ -660,4 +660,6 @@ def script_file_f():
         return ts
 
 
-stdpopsim.register_engine(_SLiMEngine())
+# SLiM does not currently work on Windows.
+if sys.platform != "win32":
```
Comment (grahamgower) [inline stdpopsim/slim_engine.py]: I suppose this should be `sys.platform.startswith("win")` to catch 64 bit windows too?

**Comment by grahamgower** [comment]
LGTM. Just a query about checking for "win32". I assume "win64" exists also?

**Comment by grahamgower** [comment]
It seems I'm fretting needlessly. sys.platform is always "win32" on windows, even on non-intel architecture ports.
https://github.com/python/cpython/blob/9052f7a41b90f2d34011c8da68f9a4facebc8a97/PC/pyconfig.h#L282

**Comment by jeromekelleher** [comment]
Yeah, it's always win32 according to the docs and [here](https://stackoverflow.com/questions/2144748/is-it-safe-to-use-sys-platform-win32-check-on-64-bit-python)

**Comment by andrewkern** [comment]
i'm running into a similar issue on #560.... why are we trying to support windows at all?

**Comment by jeromekelleher** [comment]
> i'm running into a similar issue on #560.... why are we trying to support windows at all?

Because most computers run Windows. It's elitist and exclusive to assume that people either have enough money to own a mac or are tech savvy enough to run linux. Windows support isn't that hard, and it hugely opens up the potential user base.

**Pull request merged**